### PR TITLE
Conformance test for SDX (tested by ClimateCamp)

### PR DESCRIPTION
**Pull request by 19-SD**
Contributing a Conformance Test Result for SAP Sustainability Data Exchange (SDX) tested by ClimateCamp.

### CODE REVIEW

File: catalog/conformance-tests/result-sdx-001.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "tested_solution": {
+    "solution_id": "sdx",
+    "version": "1.0.0"
```
Comment (prasadt1) [inline catalog/conformance-tests/result-sdx-001.json:4]: @19-SD - Thanks for your PR. There is one issue though - the existing version of the SDX solution in the catalog is 0.8.0 but here it says 1.0.0. Please verify!

Cc: @raimundo-henriques

Comment (19-SD) [reply catalog/conformance-tests/result-sdx-001.json:4]: @prasadt1 Nice catch! Should be 1.0.0 now. Do we update existing directory name (sdx/0.8.0) and solution file, just the solution file or create a new directory with a new file? 

I assume we will keep updating this solution version over time while remaining 'conformant'? Even though the conformance tests at some point will then refer to previous versions.

Comment (prasadt1) [reply catalog/conformance-tests/result-sdx-001.json:4]: @19-SD - Thanks for your reply. Here are 2 options we can do as PRs -

1. If SAP is Ok with keeping only the latest version i.e. v 1.0.0 for the SDX Solution, then please add a commit to update existing directory name (sdx/0.8.0) and solution file to v 1.0.0,  once done we will review the updated PR and merge it .

2. If SAP wants to keep both the versions (0.8.0 and 1.0.0), then either we or SAP can submit a new PR (or a commit in this PR), to add a new solution.json file with v 1.0.0 details. Unfortunately we haven't optimized the PACT Catalog yet to maintain version hierarchy within same Solution page, kindly note this is still a beta version and these optimisations and features are considered in the stable release when we do it. Till then if an Org has multiple versions of the Same Solution, then they will be displayed as n separate Solutions (with n different versions) on the catalog.

Please let us know what option you prefer, and accordingly submit a commit / PR or let us know we can do so as well.

Thanks
Prasad

Cc: @raimundo-henriques @zeitgeist

Comment (19-SD) [reply catalog/conformance-tests/result-sdx-001.json:4]: @prasadt1 We go for first option, solution file has been updated.

**Review by prasadt1 — APPROVED** [review]

File: catalog/conformance-tests/result-sdx-001.json (hunk 1 above)
Comment (prasadt1) [reply catalog/conformance-tests/result-sdx-001.json:4]: @19-SD - Thanks for your comment and updates. I merged the PR.

Cc: @raimundo-henriques @zeitgeist

**Pull request merged**